### PR TITLE
Fix metrics missing in data folder

### PR DESCRIPTION
**Pull request by d-shree**
Fixes for the following issues -

1. Metrics folder not found in the data/classification folders
2. Upload not working for s3, due to incompatible regions

### CODE REVIEW

File: skit_pipelines/components/evaluate_slu_from_repo/__init__.py [hunk 1]
```diff
@@ -91,8 +91,22 @@ def evaluate_slu_from_repo(
     if os.path.exists(tagged_data_path):
         final_test_dataset_path = tagged_data_path
 
-    testing(repo_name, project_config_local_path, final_test_dataset_path, remove_intents, intent_alias_path, core_slu_repo_name, comparison_classification_report_path, comparison_confusion_matrix_path, compare_branch)
-
+    classification_report_path, confusion_matrix_path = testing(repo_name, project_config_local_path, final_test_dataset_path, remove_intents, 
+                                                                intent_alias_path, core_slu_repo_name, comparison_classification_report_path, 
+                                                                comparison_confusion_matrix_path)
+    if compare:
```
Comment (harshithere) [inline skit_pipelines/components/evaluate_slu_from_repo/__init__.py:97]: Question: Why do you have this? The value of `compare` comes from `compare_branch` anyway in Line 42

File: skit_pipelines/components/upload2s3.py [hunk 2]
```diff
@@ -15,6 +15,7 @@ def upload2s3(
     output_path: str = "",
     storage_options: str = "",
     upload_as_directory: bool = False,
+    region_name: str = ""
```
Comment (harshithere) [inline skit_pipelines/components/upload2s3.py:18]: Suggestion: We dont need this as a variable. A mapping could get the `region_name` from the `bucket_name` passed. Would reduce one variable and chances of the caller making a mistake. Instead have a dict as:

```
region_mapper = {
'bucket_1': 'region_1',
'bucket_2': 'region_2'...}
```

**Review by harshithere — CHANGES_REQUESTED** [review]
Some changes suggested

File: skit_pipelines/components/upload2s3.py (hunk 2 above)
Comment (d-shree) [reply skit_pipelines/components/upload2s3.py:18]: Makes sense, will try this.

File: skit_pipelines/components/evaluate_slu_from_repo/__init__.py [hunk 3]
```diff
@@ -91,8 +91,20 @@ def evaluate_slu_from_repo(
     if os.path.exists(tagged_data_path):
         final_test_dataset_path = tagged_data_path
 
-    testing(repo_name, project_config_local_path, final_test_dataset_path, remove_intents, intent_alias_path, core_slu_repo_name, comparison_classification_report_path, comparison_confusion_matrix_path, compare_branch)
-
+    classification_report_path, confusion_matrix_path = testing(repo_name, project_config_local_path, final_test_dataset_path, remove_intents, 
+                                                                intent_alias_path, core_slu_repo_name, comparison_classification_report_path, 
+                                                                comparison_confusion_matrix_path)
+    if not compare:
```
[the remark this answers is not in the document]
Comment (d-shree) [reply skit_pipelines/components/evaluate_slu_from_repo/__init__.py]: Done

**Pull request merged**